solid-notifications-context.jsonld

```diff
@@ -0,0 +1,47 @@
+{
+  "@context": {
+    "@version": 1.1,
+    "@protected": true,
+    "id": "@id",
+    "type": "@type",
+    "notify": "http://www.w3.org/ns/solid/notifications#",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+
+    "accept": "notify:accept",
+
+    "expiration": {
+      "@id": "notify:expiration",
+      "@type": "xsd:dateTime" },
+
+    "features": {
+      "@id": "notify:features",
+      "@type": "@vocab" },
+
+    "notificationChannel": {
+      "@id": "notify:notificationChannel",
+      "@type": "@id" },
+
+    "rate": {
+      "@id": "notify:rate",
+      "@type": "xsd:duration" },
```

Comment:
[rdf-mt](https://www.w3.org/TR/rdf-mt/) (2004) discourages the use of some built-in XML Schema datatypes, e.g., `xsd:duration`.

[rdf11-mt](https://www.w3.org/TR/rdf11-mt/) (2014) does not discourage the use of `xsd:duration`.

[rdf11-concepts](https://www.w3.org/TR/rdf11-concepts/#xsd-datatypes) (2014) adds `xsd:duration` (interestingly referring to [xmlschema11-2](http://www.w3.org/TR/xmlschema11-2/#duration)) to the list of RDF-compatible datatypes (see also [rdf11-new](https://www.w3.org/TR/rdf11-new/#datatypes)).

My understanding is that the recommendations in rdf11-mt and rdf11-concepts supersede the recommendation (noting the context of the document as a WG Note) in [swbp-xsch-datatypes](https://www.w3.org/TR/swbp-xsch-datatypes/#section-duration) (2006) (interestingly referring to [xml-schema2](https://www.w3.org/TR/xmlschema-2/#duration) (2004) about rdf-mt) based on the assumption that use of xmlschema-2 is considered valid in the context of rdf11-concepts.

`xsd:duration` can be kept as part of `notify:rate`'s definition.

Reply:
Thank you for looking into this @csarven. This is very helpful.